hoomd/md/PotentialPair.h

```diff
@@ -314,6 +333,24 @@ void PotentialPair< evaluator >::setRon(unsigned int typ1, unsigned int typ2, Sc
     h_ronsq.data[m_typpair_idx(typ2, typ1)] = ron * ron;
     }
 
+template <class evaluator>
```

Comment:
I think I've usually seen `Evaluator` capitalized in other parts of the HOOMD codebase. Check for consistency.

Reply:
The `Evaluator*` classes are capitalized but the evaluator typename is lowercase everywhere, at least in `MD`.